data/mods/Xedra_Evolved/obsoletion_and_migration/spells.json

```diff
@@ -0,0 +1,249 @@
+    "spell_class": "INVENTOR",
+    "energy_source": "MANA"
+  },
+  {
+    "id": "summon_rip_ticket",
+    "type": "SPELL",
+    "name": "Make The Vancian Bill",
+    "description": "Somehow if you write some symbols on paper and cover it with mana, you can make a tool that allows you to tear a small part of reality.  You found paper dollar bills perfectly fit the required shape.  You find it hilarious that reality accepts US currency.",
+    "valid_targets": [ "self" ],
+    "min_damage": 0,
+    "max_damage": 0,
+    "effect": "attack",
+    "//": "OBSOLETED",
+    "shape": "blast",
+    "base_casting_time": 180000,
+    "final_casting_time": 24000,
+    "casting_time_increment": -7800,
+    "base_energy_cost": 450,
+    "final_energy_cost": 250,
+    "energy_increment": -10,
+    "difficulty": 4,
+    "max_level": 20,
+    "skill": "deduction",
+    "flags": [ "SOMATIC", "CONCENTRATE", "PERMANENT_ALL_LEVELS" ],
+    "spell_class": "INVENTOR",
+    "energy_source": "MANA"
+  },
+  {
+    "id": "summon_helmet_inventor",
+    "type": "SPELL",
+    "name": "Helmet of Creative Juices",
+    "description": "You know how to make a helmet that will increase your creative abilities.  Even with this improvement you don't really know how it works, it just came to you in a flash of brilliance.",
+    "valid_targets": [ "self" ],
+    "min_damage": 0,
+    "max_damage": 0,
+    "effect": "attack",
+    "//": "OBSOLETED",
+    "shape": "blast",
+    "base_casting_time": 1800000,
+    "final_casting_time": 240000,
+    "casting_time_increment": -78000,
+    "base_energy_cost": 1300,
+    "final_energy_cost": 600,
+    "energy_increment": -35,
+    "difficulty": 6,
+    "max_level": 20,
+    "skill": "deduction",
+    "flags": [ "SOMATIC", "CONCENTRATE", "PERMANENT_ALL_LEVELS" ],
+    "spell_class": "INVENTOR",
+    "energy_source": "MANA"
+  },
+  {
+    "id": "summon_ion_gun",
+    "type": "SPELL",
+    "name": "Make The Ion Rifle",
+    "description": "Craft a small weapon that does… things, that allow you to… kill… things… whatever, you have no idea how it works.",
+    "valid_targets": [ "self" ],
+    "min_damage": 0,
+    "max_damage": 0,
+    "effect": "attack",
+    "//": "OBSOLETED",
+    "shape": "blast",
+    "base_casting_time": 1800000,
+    "final_casting_time": 240000,
+    "casting_time_increment": -78000,
+    "base_energy_cost": 1400,
+    "final_energy_cost": 600,
+    "energy_increment": -40,
+    "min_duration": 302400,
+    "max_duration": 604800,
+    "duration_increment": 3024000,
+    "difficulty": 8,
+    "max_level": 20,
+    "skill": "deduction",
+    "flags": [ "SOMATIC", "CONCENTRATE" ],
+    "spell_class": "INVENTOR",
+    "energy_source": "MANA"
+  },
+  {
+    "id": "spark_of_inspiration",
+    "type": "SPELL",
+    "name": "Minor Spark of Inspiration",
+    "//": "Add two more spells one for 3 sparks and one for 5 sparks.  Also add spell components reflecting a fully stocked garage of three different tiers.",
```

Comment:
```suggestion
```
Don't need the old comment right? Maybe a new comment explaining how this is used?